pypowsybl/_pypowsybl.pyi

```diff
@@ -551,22 +552,6 @@ class FlowDecompositionParameters:
     sensitivity_variable_batch_size: int
     def __init__(self) -> None: ...
 
-
-class BranchSide:
```

Comment:
@EtienneLt  I tried to to rationalize a bit how side enum is handled and use the same enum everywhere.

Reply:
perfect